lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -646,27 +695,23 @@ void LowerLayersPass::runOnOperation() {
     }
     prefix.append(layerOp.getSymName());
 
-    auto outputFileAttr = hw::OutputFileAttr::getFromFilename(
-        builder.getContext(), prefix + ".sv",
-        /*excludeFromFileList=*/true);
-
     SmallString<128> includes;
     for (auto [_, strAttr] : layers) {
       includes.append(strAttr);
       includes.append("\n");
     }
 
     // Write header to a verbatim.
-    builder
-        .create<sv::VerbatimOp>(layerOp.getLoc(), includes + "`ifndef " +
-                                                      prefix + "\n" +
-                                                      "`define " + prefix)
-        ->setAttr("output_file", outputFileAttr);
+    setOutputFile(builder.create<sv::VerbatimOp>(
```

Comment:
Please consider using the `emit::File` and `emit::Verbatim` operations, they are active & supported.

Reply:
I can switch over to these now that they are available. Thanks!

Reply:
I'd like to handle this in a cleanup pass if that's alright.